main/lib/src/CommandReceiver.cc

```diff
@@ -112,7 +132,12 @@ namespace eudaq {
     std::cout << "Config:\n" << param << std::endl;
   }
 
-  void CommandReceiver::OnClear() { SetStatus(Status::LVL_NONE, "Wait"); }
+  void CommandReceiver::OnClear() {
+
+    if(m_connectionstate.GetState()== eudaq::ConnectionState::STATE_RUNNING)
+      OnStopRun();
```

Comment:
OnClear is used when you for example want to reconfigure your producers etc. When you press configure button this function sets your machine to UNCONF state before configuretion is completed. But I guess it can be removed if you think it is better.